docs/developer/checkout/overview.mdx

```diff
@@ -305,3 +305,24 @@ To fetch paid checkouts, use the below query:
   }
 }
 ```
+
+## Completing checkouts with free items
+
+You might have a use-case where you want to allow customers to complete a checkout without needing to pay. This can be useful for scenarios like:
+
+- Free samples
+- Free digital downloads
+
+To allow for this, you can create a checkout with a total price of `0`. This can be achieved by adding a free item to the checkout or by applying a discount code that reduces the total price to `0`.
+
+Checkout's that `totalPrice` is `0` can be completing without needing any payment, by using `checkoutComplete` mutation:
+
+```graphql
+mutation CompleteCheckout($checkoutId: ID!) {
+  checkoutComplete(id: $checkoutId) {
+    order {
+      id
+    }
+  }
+}
+```
```

Comment:
```suggestion
Checkout's that `totalPrice` is `0` can be completed without needing any payment, by using `checkoutComplete` mutation.
```

The code snipped can be skipped since it does not contain any use-case specific details.